roles/config-docker/tasks/main.yml

```diff
@@ -1,6 +1,11 @@
 ---
 
 - block:
+  - name: Configure Fedora 31 to use cgroups v1
+    import_tasks: fedora_kernel_cgroups.yml
+    when:
+    - (ansible_distribution == "Fedora") and (ansible_distribution_version == "31")
```

Comment:
As mentioned in the overall comment, I'd rather see a solution here that checks if the user requested to use `cgroup_v1` rather than if this is running on Fedora (and 31). In that way, the implementation is more generic and allows it to be applied on other OS types and versions as well.

Reply:
It was intended to be a quick fix for the Fedora 31 since we have no idea what will change in the future (Docker will adopt cgroups v2 or Fedora 3x will revert back to cgroup v1 - unlikely but you never know). It makes sense to have it for other OSes as well but this requires more research since the task uses grubby tool to make those changes which might not be available at least out of the box. Let me dig in further.